src/Client.php

```diff
@@ -143,12 +143,14 @@ public function captureLastError(?Scope $scope = null): ?string
     /**
      * {@inheritdoc}
      *
-     * @psalm-template T of IntegrationInterface
+     * @template T of IntegrationInterface
      */
     public function getIntegration(string $className): ?IntegrationInterface
     {
-        /** @psalm-var T|null */
-        return $this->integrations[$className] ?? null;
+        /** @var T|null */
+        $maybeIntegration = $this->integrations[$className] ?? null;
```

Comment:
`@psalm-var` is for "var"iables, so forcing return's type can be hard to read

the PHPStan bug is being resolved...

Reply:
Done :(

Reply:
>`@psalm-var` is for "var"iables, so forcing return's type can be hard to read

It may be hard to read, but it was suggested in vimeo/psalm#1929 as a solution to an issue similar to this one. Btw, instead of assigning a temporary variable just for the sake of having the annotation on the "right" place I prefer to leave the code as it was

>the PHPStan bug is being resolved

Do you have any link to the issue or PR to reference here for the future?